<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ECH 课堂助手</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scroll_area">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>1015</width>
      <height>620</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>966</width>
       <height>609</height>
      </rect>
     </property>
     <widget class="QPushButton" name="add_test">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>970</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>970</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>添加测试</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_stu">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>970</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>970</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>添加学生</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sort_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>970</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>按成绩排序</string>
      </property>
     </widget>
     <widget class="QLabel" name="score">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>970</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>平均分：</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QComboBox" name="choose_class">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>810</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="main">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>主菜单</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_class">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>30</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>添加班级</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>记事本</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>230</width>
      <height>620</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>991</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>该班级的 测试/学生</string>
    </property>
   </widget>
   <widget class="QPushButton" name="info">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>30</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>班级信息</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="del_class">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>30</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>删除班级</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>基础 (&amp;B)</string>
    </property>
    <addaction name="rand_stu"/>
    <addaction name="write_html"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>高级 (&amp;A)</string>
    </property>
    <addaction name="write_Python"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>设置 (&amp;S)</string>
    </property>
    <addaction name="set_font"/>
    <addaction name="set_bg"/>
    <addaction name="set_btn"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="_save">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="_main">
   <property name="text">
    <string>返回主菜单</string>
   </property>
  </action>
  <action name="set_font">
   <property name="text">
    <string>设置字体 (&amp;F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+F</string>
   </property>
  </action>
  <action name="make_svg">
   <property name="text">
    <string>绘制SVG图像 (&amp;I)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+I</string>
   </property>
  </action>
  <action name="write_Python">
   <property name="text">
    <string>编写Python代码 (&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+P</string>
   </property>
  </action>
  <action name="rand_stu">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>随机点名 (&amp;R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+R</string>
   </property>
  </action>
  <action name="write_html">
   <property name="text">
    <string>编写文档 (&amp;H)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+H</string>
   </property>
  </action>
  <action name="set_bg">
   <property name="text">
    <string>设置背景 (&amp;G)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+G</string>
   </property>
  </action>
  <action name="set_btn">
   <property name="text">
    <string>设置按钮颜色 (&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+B</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
